src/utils.ts

```diff
@@ -0,0 +1,11 @@
+export async function isUsingFirebaseJsSdk() {
+  if(!process.env.__FIREBASE_DEFAULTS__) return false;
+
+  try {
+    await import('firebase/app');
+
+    return true
+  } catch (e) {
+    return false;
+  }
+}
```

Comment:
Nit: line break at end of files.

Reply:
I see, there's no prettier config in this repo though, we should add it, probably use the same config as the CLI